### PR TITLE
Fix misc. source comment typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found using `codespell -q 3 -S ./thirdparty,*.po -L ang,ba,cas,dof,doubleclick,fave,hist,leapyear,lod,nd,numer,ois,paket,seeked,sinc,switchs,te,uint -D ~/Projects/codespell/codespell_lib/data/dictionary.txt `

### CODE REVIEW

File: modules/gdscript/gdscript.cpp [hunk 1]
```diff
@@ -99,7 +99,7 @@ GDScriptInstance *GDScript::_create_instance(const Variant **p_args, int p_argco
 #endif
 	instance->owner->set_script_instance(instance);
 
-	/* STEP 2, INITIALIZE AND CONSRTUCT */
+	/* STEP 2, INITIALIZE AND CONSTRUCTT */
```
Comment (clayjohn) [inline modules/gdscript/gdscript.cpp]: Still wrong here

File: modules/gdscript/gdscript_compiler.cpp [hunk 2]
```diff
@@ -2064,7 +2064,7 @@ Error GDScriptCompiler::_parse_class_blocks(GDScript *p_script, const GDScriptPa
 					}
 					instance->owner->set_script_instance(instance);
 
-					/* STEP 2, INITIALIZE AND CONSRTUCT */
+					/* STEP 2, INITIALIZE AND CONSTRUCTT */
```
Comment (clayjohn) [inline modules/gdscript/gdscript_compiler.cpp]: Same as above

File: modules/gdscript/gdscript.cpp (hunk 1 above)
Comment (luzpaz) [reply modules/gdscript/gdscript.cpp]: 8b30bb9

File: modules/gdscript/gdscript_compiler.cpp (hunk 2 above)
Comment (luzpaz) [reply modules/gdscript/gdscript_compiler.cpp]: 8b30bb9

**Review by clayjohn — APPROVED** [review]

**Comment by akien-mga** [comment]
Looks good, but commits should be squashed into one. See [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

**Comment by luzpaz** [comment]
Squashed.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**